extension/html/en.html

```diff
@@ -1433,7 +1432,7 @@ <h4 class="page-header">Shared Coin<br/><small>Trustless coin mixing using coin
             </div>
 
             <div class="form-actions">
-                <button class="send btn btn-primary pop" title="Review Payment" data-content="Click Here to review payment. You will be able to confirm or cancel the transaction.">Review Payment</button>
```

Comment:
<button class="send btn btn-primary pop" title="Send Payment" data-content="Click Here to send payment. You will be able to confirm or cancel the transaction.">Send Payment</button>